flake.nix

```diff
@@ -12,6 +12,8 @@
         flake-parts.follows = "flake-parts";
         crane.follows = "crane";
         rust-overlay.follows = "rust-overlay";
+        # TODO: Revert when new release of holochain is made with updated hc-sandbox cli
+        holochain.url = "github:holochain/holochain?ref=develop";
```

Comment:
Is there a tag now that we could use here so we know when Holonix has caught up? I think 0.4.0-dev.24 should have what you needed